### PR TITLE
[RW-8133][risk=no] Add ds_device to concept sets and dataset builder (ds-linking)

### PR DESCRIPTION
**Pull request by chenchals**
Description:

- Add Device to Concept Sets and Dataset Builder:

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have added explanatory comments where the logic is not obvious
- [ ] I have run and tested this change locally, and my testing process is described here
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

File: api/db-cdr/generate-cdr/build-cloudsql-tables.sh [hunk 1]
```diff
@@ -109,7 +109,8 @@ VALUES
 (3,'Domains',4,'Labs and Measurements','Labs and Measurements',0,0,1,3),
 (4,'Domains',5,'Observations','Observations',0,0,1,4),
 (5,'Domains',6,'Procedures','Procedures',0,0,1,5),
-(6,'Program Physical Measurements',19,'Physical Measurements','Participants have the option to provide a standard set of physical measurements as part of the enrollment process(program physical measurements)',0,0,0,8)"
+(6,'Program Physical Measurements',19,'Physical Measurements','Participants have the option to provide a standard set of physical measurements as part of the enrollment process(program physical measurements)',0,0,0,8),
+(7,'Devices',2,'Devices','Devices',0,0,1,9)"
```
Comment (freemabd) [inline api/db-cdr/generate-cdr/build-cloudsql-tables.sh:113]: My mistake on the ticket. Suggesting change
```suggestion
(7,'Domains',2,'Devices','Devices',0,0,1,6)"
```

File: api/db-cdr/generate-cdr/build-cloudsql-tables.sh [hunk 2]
```diff
@@ -109,7 +109,8 @@ VALUES
 (3,'Domains',4,'Labs and Measurements','Labs and Measurements',0,0,1,3),
 (4,'Domains',5,'Observations','Observations',0,0,1,4),
 (5,'Domains',6,'Procedures','Procedures',0,0,1,5),
-(6,'Program Physical Measurements',19,'Physical Measurements','Participants have the option to provide a standard set of physical measurements as part of the enrollment process(program physical measurements)',0,0,0,8)"
+(6,'Program Physical Measurements',19,'Physical Measurements','Participants have the option to provide a standard set of physical measurements as part of the enrollment process(program physical measurements)',0,0,0,8),
```
Comment (freemabd) [inline api/db-cdr/generate-cdr/build-cloudsql-tables.sh:112]: You also need to add code to update devices count. Example for updating counts for standard condition occurrence: https://github.com/all-of-us/workbench/blob/013e4226558dbc16de7a894d5d0820e08fc7efa5/api/db-cdr/generate-cdr/build-cloudsql-tables.sh#L248-L257

**Review by freemabd — COMMENTED** [review]
Couple of comments

File: api/db-cdr/generate-cdr/build-cloudsql-tables.sh (hunk 1 above)
Comment (chenchals) [reply api/db-cdr/generate-cdr/build-cloudsql-tables.sh:113]: I saw this, but thought "device" is different from "domain" :-), else I would have made the change you suggested...

File: api/db-cdr/generate-cdr/build-cloudsql-tables.sh (hunk 2 above)
Comment (chenchals) [reply api/db-cdr/generate-cdr/build-cloudsql-tables.sh:112]: got it. done

**Review by freemabd — APPROVED** [review]
Looks good.

**Pull request merged**